src/org/omegat/filters2/po/PoFilter.java

```diff
@@ -325,7 +326,7 @@ public void processFile(File inFile, File outFile, FilterContext fc) throws IOEx
 
         String disallowEditingBlankSegmentStr = processOptions.get(OPTION_ALLOW_EDITING_BLANK_SEGMENT);
         allowEditingBlankSegment = disallowEditingBlankSegmentStr == null
-                || disallowEditingBlankSegmentStr.equalsIgnoreCase("true");
+                || disallowEditingBlankSegmentStr.equalsIgnoreCase("false");
```

Comment:
Instead of inverting this, we should invert `disallowEditingBlankSegmentStr` (change it to `allowEditingBlankSegmentStr` and adjust all logic to match) because everywhere else in the program the setting is phrased as a positive ("allow"), not a negative ("disallow").

Reply:
I've changed it to ``allowEditingBlackSegmentStr`` and keep it to compare with "true" as same as the suggestion.